### PR TITLE
[6] [7] - Setup Terraform Cloud and the Base Terraform codebase

### PR DESCRIPTION
**Pull request by andyduong1920**
https://github.com/nimblehq/nimble-growth-39-k8s-aws-eks/issues/6
https://github.com/nimblehq/nimble-growth-39-k8s-aws-eks/issues/7

## What happened 👀

- Setup the Terraform Cloud
- Generate the base Terraform codebase - without the ECS module from the nimble infra template

## Insight 📝

- Using Nimble Infra Template
- Add the CodeOwner file
- Add the Github Action lint

## Proof Of Work 📹

On the file changes tab

1/ Terraform init

image

2/ Terrafirm apply
image

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Team
+* @andyduong1920 @hoangmir @junan @longnd @nvminhtue
```
Comment (hoangmirs) [inline .github/CODEOWNERS:2]: Please correct my name 🙏 
```suggestion
* @andyduong1920 @hoangmirs @junan @longnd @nvminhtue
```

File: .github/workflows/linting.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    timeout-minutes: 5
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v2
```
Comment (hoangmirs) [inline .github/workflows/linting.yml:23]: We can upgrade this to use v3:
```suggestion
        uses: actions/checkout@v3
```

Comment (andyduong1920) [reply .github/workflows/linting.yml]: Fixed in https://github.com/nimblehq/nimble-growth-39-k8s-aws-eks/pull/11/commits/b9bbc7d7b24240d5064d3112d6be84f575503143 @hoangmirs

File: .github/CODEOWNERS (hunk 1 above)
Comment (andyduong1920) [reply .github/CODEOWNERS]: Oh sorry my bad, I have fixed it in https://github.com/nimblehq/nimble-growth-39-k8s-aws-eks/pull/11/commits/b9bbc7d7b24240d5064d3112d6be84f575503143

**Review by nvminhtue — APPROVED** [review]

**Review by hoangmirs — APPROVED** [review]
LGTM

**Pull request merged**